docs/aurora/running-jobs-aurora.md

```diff
@@ -449,6 +449,27 @@ This command requests 1 node for a period of 1 hour in the `workq` queue. After
 **NOTE:** If you want to `ssh` or `scp` to one of your assigned compute nodes you will need to make sure your `$HOME` directory and your `$HOME/.ssh` directory permissions are both set to `700`.
 
 
+## <a name="Running-with-Multiple-CCS"></a>Running with Multiple Compute Command Streamers (CCSs)
```

Comment:
even though you didnt add it, could you turn the above NOTE into an admonition, if appropriate?